lib/sessions.ts

```diff
@@ -85,6 +109,98 @@ export class Sessions {
     });
   }
 
+  /** Parse a JWT and verify the signature, preferring local verification over remote.
+   *
+   * If maxTokenAge is set, remote verification will be forced if the JWT was issued at (based on
+   * the "iat" claim) more than maxTokenAge seconds ago.
+   */
+  async authenticate_jwt(
+    jwt: string,
+    options?: {
+      maxTokenAge?: number; // seconds
+    }
+  ): Promise<AuthenticateResponse> {
+    try {
+      const session = await this.authenticate_jwt_local(jwt, options);
+      return {
+        session,
+        session_jwt: jwt,
+        session_token: "",
+
+        // Pretend that this was a successful API request.
+        request_id: "",
+        status_code: 200,
+      };
+    } catch (err) {
+      if (err instanceof ClientError && err.code === "jwt_too_old") {
+        // Token was too old (stale) to verify locally. Check with the Stytch API.
```

Comment:
```suggestion
        // JWT was too old (stale) to verify locally. Check with the Stytch API.
```
Very much a nit, but prefer JWT here to "token" since our API has endpoints that accept a "token" value and that is not a JWT.

Reply:
... I have no explanation for why I said "token" here. Thanks for catching it!